src/lib/i18n/locales/en/workflows.ts

```diff
@@ -90,6 +90,8 @@ export const Strings = {
   'reset-event-radio-group-description': 'Choose an Event to reset to',
   'reset-reapply-type-label':
     'Reapply Signals that happened after the Reset point',
+  'reset-exclude-signals': 'Exclude signals that occur after the reset point.',
+  'reset-exclude-updates': 'Exclude updates that occur after the reset point.',
```

Comment:
Let's capitalize Signals and Updates here

Reply:
and Reset too.